packages/kbn-alerts-as-data-utils/src/field_maps/alert_field_map.ts

```diff
@@ -82,6 +83,11 @@ export const alertFieldMap = {
     array: false,
     required: false,
   },
+  [ALERT_CONTEXT]: {
```

Comment:
this adds `kibana.alert.context` to the framework alert mappings that applies to all alerts as data docs, not just the log threshold mappings. was that the intent?

Reply:
If you just want it for the log threshold rule type, I would maybe just update the `legacyExperimentalFieldMap` in `packages/kbn-alerts-as-data-utils/src/field_maps/legacy_experimental_field_map.ts`. That is also used by the other observability rule types but doesn't affect every rule type.

Reply:
Yes...  we plan to update ALL of the Observability Rules to store their context in the AAD documents. There have been few things we've needed access to for the Alert Details page. Every time we need something from the context, we have to debate "Where should we store that in AAD?" This seems like something every alert document should have, the context the alert was triggered with.

Reply:
I'll move it to the legacy_experimental_field_map.ts